plugins/vpp/l3plugin/vppcalls/vpp1908/route_vppcalls.go

```diff
@@ -114,6 +109,56 @@ func (h *RouteHandler) VppDelRoute(route *vpp_l3.Route) error {
 	return h.vppAddDelRoute(route, swIfIdx, true)
 }
 
+func setFibPathNhAndProto(ipStr string) (nh ip.FibPathNh, proto ip.FibPathNhProto) {
+	netIP := net.ParseIP(ipStr)
+	if netIP == nil {
+		return
+	}
+	var ipData [16]byte
+	if netIP.To4() == nil {
+		proto = ip.FIB_API_PATH_NH_PROTO_IP6
+		copy(ipData[:], netIP[:])
+	} else {
+		proto = ip.FIB_API_PATH_NH_PROTO_IP4
+		copy(ipData[:], netIP[12:])
+	}
+	return ip.FibPathNh{
+		Address: ip.AddressUnion{
+			XXX_UnionData: ipData,
+		},
+		ViaLabel:           NextHopViaLabelUnset,
+		ClassifyTableIndex: ClassifyTableIndexUnset,
+	}, proto
+}
+
+func setRoutePrefix(dstNetwork string) (ip.Prefix, error) {
```

Comment:
The name of this function does not really define what it's doing, it's named after it's only use.

The function actually parses CIDR string into `ip.Prefix` and should be named accordingly and placed next to other [general parsing function in `vppcalls_handlers.go`](https://github.com/ligato/vpp-agent/blob/dev/plugins/vpp/l3plugin/vppcalls/vpp1908/vppcalls_handlers.go#L182) so it can be used by other functions when needed.